fe/fe-core/src/main/java/org/apache/doris/analysis/UseStmt.java

```diff
@@ -35,19 +35,35 @@
  */
 public class UseStmt extends StatementBase {
     private static final Logger LOG = LogManager.getLogger(UseStmt.class);
+    private String catalogName;
     private String database;
 
     public UseStmt(String db) {
         database = db;
     }
 
+    public UseStmt(String catalogName, String db) {
```

Comment:
No need to modify use stmt in `sql_parser.cup`?

Reply:
Still in working